common/types/block.go

```diff
@@ -11,16 +11,21 @@ import (
 	"sort"
 )
 
+// AtxsPerBlockLimit indicates the maximum number of atxs a block can reference
+const AtxsPerBlockLimit = 100
+
 // BlockID is a 20-byte sha256 sum of the serialized block, used to identify it.
 type BlockID Hash20
 
 // String returns the 5-character prefix of the hex representation of the ID.
 func (id BlockID) String() string {
-	return id.AsHash32().ShortString()
+	return id.AsHash32().MediumString()
 }
 
 // Field returns a log field. Implements the LoggableField interface.
-func (id BlockID) Field() log.Field { return log.String("block_id", id.AsHash32().ShortString()) }
+func (id BlockID) Field() log.LoggableField {
```

Comment:
because you changed the return type from `log.Field` to `log.LoggableField` here, `BlockID` now no longer implements the `log.LoggableField` interface and therefore can't be used as a field directly...

You've made this change in many places and I don't understand the motivation for it...

Worth noting that `log.Field` itself implements the `log.LoggableField` interface, so it should work for most use-cases the way it was.